src/Features/Core/Portable/ExternalAccess/VSTypeScript/Api/VSTypeScriptSignatureHelpProviderBase.cs

```diff
@@ -14,11 +16,50 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api;
 /// </summary>
 internal abstract class VSTypeScriptSignatureHelpProviderBase : ISignatureHelpProvider
 {
+    public ImmutableArray<char> TriggerCharacters
```

Comment:
this was the easiest way I could come up with to avoid a dual insertion.

once we have packages I'll submit a PR to TS to switch to the new properties, then we can delete the hardcoded characters.